src/System.Reflection.Emit.ILGeneration/tests/CustomAttributeBuilderTests.cs

```diff
@@ -803,20 +804,15 @@ public static void ZeroCountMultidimensionalArrayInFieldValues_ChangesToZeroCoun
         }
 
         [Theory]
-        [MemberData(nameof(NotSupportedPrimitives_CustomAttributeFormatException_TestData))]
-        public static void NotSupportedPrimitiveInFieldValues_ThrowsCustomAttributeFormatExceptionOnCreation(object value)
+        [MemberData(nameof(NotSupportedPrimitives_TestData))]
```

Comment:
This looks like a breaking change on inspection. As I recall, the original version was asserting inside EmitType() and not writing out anything for CustomAttributeEncoding, thus creating a blob that will (probably) generate a CustomAttributeFormatException, but could in theory generate something more uncontrolled.

A comment detailing this here would be good so we'll remember why we made this change.